pymesync.py

```diff
@@ -0,0 +1,70 @@
+"""
+pymesync - Python TimeSync Module
+
+Allows for interactions with the TimeSync API
+- timesync.send_time(baseurl) -- Sends time to baseurl (TimeSync)
+"""
+import json
+import requests
+
+
+class TimeSync(object):
+
+    def __init__(self, baseurl, user, password, auth_type):
+        self.baseurl = baseurl
+        self.user = user
+        self.password = password
+        self.auth_type = auth_type
+
+    def send_time(self, parameter_dict):
+        """
+        send_time(parameter_dict)
+
+        Sends a POST request in a JSON body to TimeSync, returns that body if
+        successful or an error if not.
+        parameter_dict - python dict containing time info.
+        """
+
+        values = {
+            'auth': self._auth(),
+            'object': {
+                'duration': parameter_dict['duration'],
+                'user': parameter_dict['user'],
+                'project': parameter_dict['project'],
+                'activities': parameter_dict['activities'],
+                'date_worked': parameter_dict['date_worked'],
+                'notes': parameter_dict['notes'],
+                'issue_uri': parameter_dict['issue_uri'],
```

Comment:
I like the example you put up. That makes it a lot cleaner. The issue with having default values as secondary arguments is that there isn't a good way to pick the defaults. All of that could change and probably should change every post. Maybe ``date_worked`` could be set to the current date but that is the only one that makes sense to me.